### ISSUE TITLE
tests with wirtual test cases are harder to move

### ISSUE THREAD

**Issue by sopos** [psss/fmf#81]
Currently, if a test referenced by multiple test cases is imported few files .fmf are created.
the `main.fmf` contains `path` key so other `something.fmf` inherit it and not expand it on their own.
If such test case is moved to another location the path need to be updated as well.
It would be nice if there was a way to use relative references, e.g. in the virtual cases use something like `path: ..` or in main.fmf use `path: .`

**Comment by psss** [comment]
I believe this use case should be resolved by fixing psss/tmt#239. Seems to be the same scenario. There we propose that `path` should default to the directory where metadata are stored and @lukaszachy is going to implement this. So for virtual cases that path could be omitted as well and thus they can be easily moved to another location. Does that cover your use case?

**Comment by sopos** [comment]
Yeah, it look that's it. So do I understand correctly that the _path_ key is solely on `tmt` side?

**Comment by psss** [comment]
Yes, `fmf` does not know anything about what particular field is used for. Moving the issue under `tmt`.
